js/calculator.js

```diff
@@ -1,155 +1,267 @@
+            screenValue.value = n;
+            break;
     }
 }
-//six function
-const six = () =>{
-    if(document.querySelector("#display"). value == " "){
-        document.querySelector("#display").value = "6";
-    }
-    else{
-        document.querySelector("#display").value = document.querySelector("#display").value + "6";
+//function 
+function fun(num)
+{
+    switch(num)
+    {
+        case 'abslute':
+            screenValue.value = Math.abs(screenValue.value);
+            break;
+        case 'squrX':
+            screenValue.value = '('+screenValue.value +')';
+            break;
+        case 'dolor':
+            screenValue.value = screenValue.value*81.60 + " Rs" ;
+            break;
     }
 }
-//seven function
-const seven = () =>{
-    if(document.querySelector("#display"). value == " "){
-        document.querySelector("#display").value = "7";
-    }
-    else{
-        document.querySelector("#display").value = document.querySelector("#display").value + "7";
+//memory function
+function memoryOperation(str){   
+    switch(str){
+        case 'mc':
+            screenValue.value = localStorage.clear();
+            break;
+        case 'mAdd':
+            handleMPlus();
+            break;
+        case 'mSubtract':
+            handleMSubtract();
+            break;
+        case 'mr':
+            mrHandle();
+            break;
+        case 'ms':
+            localStorage.setItem("memoryValue",JSON.stringify(memoryArray));
+            break;
     }
 }
-//eight function
-const eight = () =>{
-    if(document.querySelector("#display"). value == " "){
-        document.querySelector("#display").value = "8";
-    }
-    else{
-        document.querySelector("#display").value = document.querySelector("#display").value + "8";
-    }
+//handle M+
+function handleMPlus(){  
+    sign = "Plus";
+    let result = screenValue.value;
+    memoryArray.push(result); 
 }
-//nine function
-const nine = () =>{
-    if(document.querySelector("#display"). value == " "){
-        document.querySelector("#display").value = "9";
-    }
-    else{
-        document.querySelector("#display").value = document.querySelector("#display").value + "9";
-    }
+//handle M-
+function handleMSubtract(){  
+    sign = "Minus";
+    let result = screenValue.value;
+    memoryArray.push(result); 
 }
-//zero function
-const zero = () =>{
-    if(document.querySelector("#display"). value == " "){
-        document.querySelector("#display").value = "0";
-    }
-    else{
-        document.querySelector("#display").value = document.querySelector("#display").value + "0";
-    }
+//handle MR
+function mrHandle(){
+    let ans=Number.parseInt(memoryArray[0]);
+    for(let i = 1; i < memoryArray.length; i++){
+        if(sign=="Plus")
+        {
+            ans = ans + Number.parseInt(memoryArray[i]);
+        }
+        if(sign=="Minus")
+        {
+            ans = ans - Number.parseInt(memoryArray[i]);
```

Comment:
You can parseInt once with store valu in different variable and then you can use it to avoid again parseInt